Tempo/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@perses-dev/tempo",
   "private": true,
-  "version": "0.1.0",
+  "version": "0.1.1",
```

```suggestion
"version": "0.2.0",
```
same reason than for Prometheus, it's not a patch